docs/source/cdk/aws-setup.rst

```diff
@@ -0,0 +1,45 @@
+AWS Setup
+=========
+Download/requirements
+~~~~~~~~~~~~~~~~~~~~~
+
+Ensure you have installed nodejs (newer than version 14), AWS CLI, and Docker
+
+- `nodejs <https://nodejs.org/en/download/>`_
+- `AWS CLI <https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html>`_
+- `Docker <https://docs.docker.com/get-docker/>`_
+
+.. _aws-new-user:
+New User
```

Comment:
```suggestion
.. _aws-new-user:

New User
```
I'd add a newline here as well just to immediately call out this reference link.